lib/ipmi_backend_utils.pm

```diff
@@ -21,28 +21,45 @@ use version_utils qw(is_storage_ng is_sle);
 use utils;
 use power_action_utils 'prepare_system_shutdown';
 
-our @EXPORT = qw(set_serial_console_on_vh switch_from_ssh_to_sol_console);
+our @EXPORT = qw(set_serial_console_on_vh switch_from_ssh_to_sol_console use_sol_serial_console do_ipmi_mc_reset);
 
 #With the new ipmi backend, we only use the root-ssh console when the SUT boot up,
 #and no longer setup the real serial console for either kvm or xen.
 #When needs reboot, we will switch back to sut console which relies on ipmi.
 #We will mostly rely on ikvm to continue the test flow.
 #TODO: we need the serial output to debug issues in reboot, coolo will help add it.
 
+sub use_sol_serial_console {
+    set_var('SERIALDEV', '');
+    $serialdev = 'ttyS1';
+    bmwqemu::save_vars();
+    select_console 'sol', await_console => 0;
+    save_screenshot;
+}
```

Comment:
@coolo to me it seems like with the virtio/serial/ipmi consoles we have more and more wrappers where instead the code should be in either the console activation or initialisation so that from the test code a simple `select_console 'sol'` should still be enough, WDYT?